logistics_project/apps/malawi/warehouse/report_views/stock_status.py

```diff
@@ -156,7 +156,7 @@ def get_stockout_report(self, request, reporting_supply_point, selected_type, is
         }
         stockout_data = []
         active = Product.objects.filter(is_active=True, type__is_facility=is_facility)
-        products = active.objects.filter(type=selected_type) if selected_type else active
+        products = active.filter(type=selected_type) if selected_type else active
```

Comment:
lol, wow. I just tried and got `It looks like there's no data for your filters. You've been redirected.` so maybe no one has bothered reporting it all the way up to us.